apiary.apib

```diff
@@ -518,6 +518,28 @@ Get a list of all the definitions of custom fields.
 
     + Attributes (object)
         + page (Page, optional)
+        + filter (object, optional)
+            + context (enum[string], optional)
+                + Members
+                    + contact
+                    + company
+                    + form
+                    + event
+                    + product
```

Comment:
PP : this is not an existing type as far as the API is concerned
we still need to add endpoints for products, so it doesn't make sense to have the custom fields for it